src/bliss/bench_ART.h

```diff
@@ -0,0 +1,49 @@
+#ifndef BLISS_BENCH_ART
+#define BLISS_BENCH_ART
+
+#include <ART.h>
+
+#include <vector>
+
+#include "bliss/bliss_index.h"
+#include "spdlog/spdlog.h"
+
+
+namespace bliss {
+
+template <typename KEY_TYPE, typename VALUE_TYPE>
+class BlissARTIndex : public BlissIndex<KEY_TYPE, VALUE_TYPE> {
+   public:
+   ART::Node* _index;
+    BlissARTIndex() {
+        _index = nullptr;
+    };
+
+    void bulkload(
+        std::vector<std::pair<KEY_TYPE, VALUE_TYPE>> values) override {
+        // expects the pairs to be pre-sorted before performing bulk load
+        for (const auto& pair : values) {
+            put(pair.first, pair.second);
+        }
+    }
+
+    bool get(KEY_TYPE key) override { 
+        uint8_t ARTkey[8];
+        ART::loadKey(key, ARTkey);
+
+        ART::Node* leaf = ART::lookup(_index, ARTkey, 8, 0, 8);
```

Comment:
let's use formal member variables for the key sizes and offsets (i.e., "8, 0, 8") so that our code is clear.

Reply:
Should I use enums for this?

Reply:
enums might be an overkill here... if we have a member variable for key size, then we can use that right?

Reply:
OOOHH I see that's very smart, I'll work on that right now.